routes/assets-route.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const express = require('express');
+const multer = require('multer');
+const AssetHandler = require('../services/asset-handler');
+
+const upload = multer({ dest: 'uploads/' });
+
+const router = express.Router();
+
+/**
+ * Upload a number of images to the server
+ */
+router.post('', upload.array('images', 13), async (req, res) => {
```

Comment:
why 13?

Reply:
no particular reason except on my laptop 13 is two rows of images on the client. I wanted to limit it to some arbitrary number because between processing and uploading this could turn pretty long

Reply:
Make 13 a const at the top of the file then?